### ISSUE TITLE
Build Failure: Missing Dependencies for flutter_serial_communication Due to usb-serial-for-android Version 3.5.1 Resolution Issues

### ISSUE THREAD

**Issue by HristoPetkov94** [farellsujanto/flutter_serial_communication#18]
```
Execution failed for task ':flutter_serial_communication:generateReleaseRFile'.
> Could not resolve all files for configuration ':flutter_serial_communication:releaseCompileClasspath'.
   > Failed to transform usb-serial-for-android-3.5.1.aar (com.github.mik3y:usb-serial-for-android:3.5.1).
      > Could not find usb-serial-for-android-3.5.1.jar (com.github.mik3y:usb-serial-for-android:3.5.1).
        Searched in the following locations:
            https://jitpack.io/com/github/mik3y/usb-serial-for-android/3.5.1/usb-serial-for-android-3.5.1.aar
            https://jitpack.io/com/github/mik3y/usb-serial-for-android/3.5.1/usb-serial-for-android-3.5.1.jar

```

**Comment by purnawanyose** [comment]
my resolve for this problem is pull the project / download as zip. inside project android gradle change the version for usb serial for android to 3.8.0. then back to root folder., notice example folder, look pubspec.yaml. you'll see flutter_serial_communication: ../ . my resolve is like the example folder.. so i build the app with local library.. no issue

**Comment by DominikStarke** [comment]
Can you please test my pull request? I updated the library to 3.8.0 already:
https://github.com/farellsujanto/flutter_serial_communication/pull/17

**Comment by purnawanyose** [comment]
thanks, running well

**Comment by farellsujanto** [comment]
released v0.2.7 with the fix, closing this issue
